src/api/app/controllers/trigger_workflow_controller.rb

```diff
@@ -60,4 +61,11 @@ def create_workflow_run
     request_headers = request.headers.to_h.keys.map { |k| "#{k}: #{request.headers[k]}" if k.match?(/^HTTP_/) }.compact.join("\n")
     @workflow_run = @token.workflow_runs.create(request_headers: request_headers, request_payload: request.body.read)
   end
+
+  def check_supported_scm_event
+    @scm_webhook = TriggerControllerService::ScmExtractor.new(scm, event, payload).call
+
+    # We don't want to return error to SCM for the events that are not supported.
+    render_ok if @scm_webhook.present? && !@scm_webhook.allowed_pull_request_or_merge_request_event?
```

Comment:
After discussing offline if we can simply use `!@scm_webhook.valid?` here, we concluded that we can't because the validator not only validates events for pull request, but also validates push events (see code [here](https://github.com/openSUSE/open-build-service/blob/e6ce39895e7f8e9a51d0134868206826fea857ef/src/api/app/validators/scm_webhook_event_validator.rb#L51)). And we don't plan to ignore the errors related to `push` for now.